Formula/squashfs.rb

```diff
@@ -6,6 +6,14 @@ class Squashfs < Formula
   license "GPL-2.0"
   head "https://github.com/plougher/squashfs-tools.git"
 
+  # upstream adds `4.4-git.1` tag as prerelease tag for testing,
+  # which disrupts livecheck flow, thus adding livecheck block in here
+  # see discussions in here, https://github.com/plougher/squashfs-tools/issues/96
```

Comment:
```suggestion
  # Tags like `4.4-git.1` are not release versions and the regex omits these
  # (see: https://github.com/plougher/squashfs-tools/issues/96).
```

The standard regex for Git tags (i.e., `/^v?(\d+(?:\.\d+)+)$/i`) implies that we're omitting tags that aren't like 1.2.3/v1.2.3, so we can make the explanatory comment a bit more concise.